spec/controllers/shared_context.rb

```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
+  let(:user) { create(:user) }
```

Comment:
I prefer to call logged user as current_user
```suggestion
  let(:current_user) { create(:current_user) }
```